json/field/field.go

```diff
@@ -0,0 +1,158 @@
+package field
```

Comment:
We should probably move the entire json package under the expression mapper for now.  Its not used by anything else at the moment.  We can move it up if it become more integral

Reply:
Sure. I also checked with @mellistibco .  I will move it to expression mapper for now